### DISCUSSION TITLE
MAX32655FTHR# Support?

### DISCUSSION THREAD

**Discussion by KipIngram** [ARMmbed/DAPLink#1036] [General]
Hi folks.  So, I'm interested in the MAX32655FTHR#:

https://www.digikey.com/en/products/detail/analog-devices-inc-maxim-integrated/MAX32655FTHR/17885194

At the daplink.io site I don't see the 32655 in the list of supported units, but the documentation for the item here:

https://www.analog.com/media/en/technical-documentation/data-sheets/MAX32655FTHR.pdf

says in the  "Programming and Debugging" section that DapLink can be used to flash it.

Am I looking in the wrong place?  Advice would be appreciated.  I'm interested in the simplest possible toolchain that will let me flash this board.  I'm eventually looking to write all my code myself, in the context of a Forth system that I plan to implement for it, so I don't need a development environment that all tied in with a bunch of existing libraries or anything.

Thanks in advance!!!
Kip

**Comment by mathias-arm** [comment]
The images for the MAX32625PICO are available at https://github.com/MaximIntegrated/max32625pico-firmware-images

**Reply by KipIngram** [reply]
Is that somehow "equivalent" to the product I linked above?  I saw 32625 image available on the daplink.io site, but not 32655, which is my part #.

**Reply by mathias-arm** [reply]
`MAX32625PICO` is the probe not the target. The kits I have were shipped with it, but I don't know if it true for all of them. You can program it to support different targets, `MAX32655FTHR` being one of them. The images on https://daplink.io are old versions (v254), new binaries are distributed in the other repository.

**Reply by KipIngram** [reply]
Ok, I see now - I looked more closely at the documentation - there is a 32625 on this board, in addition to a 32655 - they are separate line items on the BOM.  And both are mentioned in the "Programming and Debugging" blurb.  I just didn't read carefully enough before.

Well, that kind of answers my question, and now I feel somewhat silly.  Thanks very much, Mathias.  Take care!
